packages/expo-cli/src/appleApi/authenticate.ts

```diff
@@ -7,6 +11,99 @@ import prompt from '../prompt';
 import { nonEmptyInput } from '../validators';
 import { runAction, travelingFastlane } from './fastlane';
 
+const NO_STORE_PASSWORD = getenv.boolish('EXPO_NO_STORE_PASSWORD', false);
+
+function getKeychainServiceName(appleId: string): string {
```

Comment:
Document this clearly and link to where Fastlane/Deliver documents their naming scheme. There is coupling between this code and Fastlane that needs to be made obvious.